README.md

```diff
@@ -586,6 +626,7 @@ To update kdb VS Code settings, search for **kdb** from _Preferences_ > _Setting
 | **Hide subscription to newsletter after first install**        | yes/no; default no                                                  |
 | **Insights Enterprise Connections for Explorer**               | [edit JSON settings](#insights-enterprise-connections-for-explorer) |
 | **Linting**                                                    | Enable linting for q and quke files                                 |
+| **Refactoring**                                                | Choose refactoring scope                                            |
```

Comment:
Do we need any documentation on this new option?

Reply:
Added refactoring doc.